osgp/protocol-adapter-dlms/osgp-dlms/src/test/java/org/opensmartgridplatform/dlms/services/ObjectConfigServiceTest.java

```diff
@@ -165,18 +155,37 @@ void testGetCosemObjectsDsmr422() throws ObjectConfigException {
         this.objectConfigService.getCosemObjects("DSMR", "4.2.2");
 
     assertNotNull(cosemObjects);
-    assertThat(cosemObjects).hasSize(43);
+    AssertionsForInterfaceTypes.assertThat(cosemObjects)
+        .hasSize(Protocol.DSMR_4_2_2.getNrOfCosemObjects());
+    assertNotNull(cosemObjects.get(DlmsObjectType.ALARM_REGISTER_1));
     assertNotNull(cosemObjects.get(DlmsObjectType.NUMBER_OF_VOLTAGE_SWELLS_FOR_L1));
     assertNull(cosemObjects.get(DlmsObjectType.CDMA_DIAGNOSTIC));
   }
 
+  @Test
+  void getCosemObjectIsNull() throws ObjectConfigException {
+    final String protocolName = "SMR";
+    final String protocolVersion43 = "4.3";
+
+    final Map<DlmsObjectType, CosemObject> cosemObjects =
+        this.objectConfigService.getCosemObjects(protocolName, protocolVersion43);
+
+    assertNotNull(cosemObjects);
+    AssertionsForInterfaceTypes.assertThat(cosemObjects)
+        .hasSize(Protocol.SMR_4_3.getNrOfCosemObjects());
+    assertNotNull(cosemObjects.get(DlmsObjectType.ALARM_REGISTER_1));
+    assertNull(cosemObjects.get(DlmsObjectType.MBUS_DIAGNOSTIC));
+  }
+
```

Comment:
This test seems to have the same purpose as the next test.

Reply:
Indeed, merged these two tests.